pkg/resource/google/google_bigquery_dataset_test.go

```diff
@@ -18,10 +18,15 @@ func TestAcc_Google_BigqueryDataset(t *testing.T) {
 		},
 		Checks: []acceptance.AccCheck{
 			{
-				// New resources are not visible immediately on GCP api after an apply operation
-				// Logic below retry driftctl scan until we can retrieve the results (infra will be in sync) and for maximum 60 seconds
+				// New resources are not visible immediately through GCP API after an apply operation.
+				// Logic below retries driftctl scan using a back-off strategy of retrying 'n' times
+				// and doubling the amount of time waited after each one.
 				ShouldRetry: func(result *test.ScanResult, retryDuration time.Duration, retryCount uint8) bool {
```

Comment:
if we stop using retryCount, and instead retry however many times it takes to fill the max duration (10m in this PR), should we eliminate the parameter and follow the compiler errors?

```suggestion
				ShouldRetry: func(result *test.ScanResult, retryDuration time.Duration) bool {
```

Is there any reason we can't do this?